airbyte-integrations/connectors/source-klaviyo/source_klaviyo/source.py

```diff
@@ -58,7 +58,7 @@ def streams(self, config: Mapping[str, Any]) -> List[Type[Stream]]:
         return [
             Campaigns(api_key=config.api_key),
             Events(api_key=config.api_key, start_date=config.start_date),
-            GlobalExclusions(api_key=config.api_key),
+            GlobalExclusions(api_key=config.api_key, start_date=config.start_date),
             Lists(api_key=config.api_key),
```

Comment:
@keu please enable `Metrics` stream? See PR https://github.com/airbytehq/airbyte/pull/8673 I'll close in favor of yours.

Reply:
@marcosmarxm done, I left configured_catalog.json unchanged because SAT is not using it anymore